src/app/public/modules/file-attachment/file-attachment.component.scss

```diff
@@ -0,0 +1,57 @@
+@import "~@skyux/theme/scss/mixins";
+@import "~@skyux/theme/scss/_compat/mixins";
+
+.sky-file-drop-required {
+  box-shadow: 0 0 8px rgba(239, 64, 68, 0.6);
+  border: 1px solid #ef4044;
+}
+
+.sky-file-input-hidden {
+  display: none;
+}
+
+.sky-single-file-wrapper {
+  display: inline-block;
+}
+
+.sky-single-file-upload {
+  padding-right: 8px;
+  display: block;
+}
+
+.sky-file-name {
+  padding: 0 5px 0 5px;
+}
+
+.sky-single-file-drop-accept {
+  border-color: $sky-highlight-color-success;
+  border-style: dashed;
+  border-width: 1px;
+  border-radius: 5px;
+  color: $sky-text-color-default;
+}
+
+.sky-single-file-drop-reject {
+  border-color: $sky-highlight-color-danger;
+  border-style: dashed;
+  border-width: 1px;
+  border-radius: 5px;
+  color: $sky-text-color-default;
+}
+
+.sky-file-delete {
+  border: 0;
+  width: 24px;
+  height: 24px;
+  padding: 0;
+  margin: 0;
+  color: black;
+  cursor: pointer;
+}
+
+.sky-file-item-preview-img {
+  max-width: 25%;
+  height: auto;
+  box-shadow: 0 0 5px #666;
+  margin: 10px;
+}
```

Comment:
A general question about all the classes here - 
I noticed that we're using similar class names as the `<sky-file-drop>` component. Even though Angular will scope these classes to the specific component they fall under, I'm concerned that it causes unnecessary confusion to a developer sifting through the code. Should these all have their own prefix of `sky-file-attachment-` (or something similar)?

Reply:
Good point. I didn't do a great job cleaning up the class names after splitting the single file attachment from the normal file-drop component. I'll make these more consistent.